docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -230,7 +230,7 @@ Following recommendation apply:
   from one machine to the other.
 - Keep machine labels and annotation stable, because changing those values requires machines rollouts;
   also, please note that machine labels and annotation are not propagated to Kubernetes nodes; see
-  [metadata propagation](../../../developer/architecture/controllers/metadata-propagation.md).
+  [metadata propagation](../../../reference/api/metadata-propagation.md).
```

Comment:
I'm fixing the link, but the entire content of this sections seems outdated, I will send a follow-up PR to fix it